crates/core/src/test_utils/mod.rs

```diff
@@ -65,9 +65,13 @@ impl TestTables {
         }
     }
 
-    pub fn table_builder(&self) -> DeltaTableBuilder {
-        let url = Url::from_directory_path(self.as_path()).unwrap();
-        DeltaTableBuilder::from_uri(url).with_allow_http(true)
+    pub fn table_builder(&self) -> Result<DeltaTableBuilder, crate::DeltaTableError> {
```

Comment:
the fact the getting a builder is fallible seems counter-intuitive, given that this is a testing-only thing though its fine - unwrapping like we did before certainly is not better :D.